weldx/asdf/file.py

```diff
@@ -858,14 +858,18 @@ def show_asdf_header(
             Should not be set.
         """
         return _HeaderVisualizer(self._asdf_handle).show(
-            use_widgets=use_widgets, path=path, _interactive=_interactive
```

Comment:
Is there a specific reason you removed the `tag` argument here? @marscher 

Did I pout it here by mistake?

Reply:
my bad.

Reply:
and good catch, it seems this new option lacks a test.

Reply:
maybe .. 👼